packages/next/migrations.json

```diff
@@ -200,6 +200,19 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "13.3.0": {
+      "version": "13.3.0-beta.0",
```

Comment:
This is because Nx v13.3.0 supports the latest version of `angular-eslint` and I expect 
 it to be shipped soon. https://github.com/nrwl/nx/commit/512f094f986a67704b5e703ac1c06fdac71ff6bb

Reply:
Next.js v12.0.5+ supports ESLint v8, which means this migration is suitable for Nx v13.3.0